src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs

```diff
@@ -371,6 +468,70 @@ public bool TryGetBytesFromBase64(out byte[] value)
             return JsonReaderHelper.TryDecodeBase64(span, out value);
         }
 
+        /// <summary>
+        /// Parses the current JSON token value from the source as a <see cref="byte"/>.
+        /// Returns true if the entire UTF-8 encoded token value can be successfully 
+        /// parsed to a <see cref="byte"/> value.
+        /// Returns false otherwise.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
+        /// Thrown if trying to get the value of a JSON token that is not a <see cref="JsonTokenType.Number"/>.
+        /// <seealso cref="TokenType" />
+        /// </exception>
+        public bool TryGetByte(out byte value)
+        {
+            if (TokenType != JsonTokenType.Number)
+            {
+                throw ThrowHelper.GetInvalidOperationException_ExpectedNumber(TokenType);
+            }
+
+            ReadOnlySpan<byte> span = HasValueSequence ? ValueSequence.ToArray() : ValueSpan;
+            return Utf8Parser.TryParse(span, out value, out int bytesConsumed) && span.Length == bytesConsumed;
```

Comment:
Check out #37838 for how these internal TryParse calls should be structured.

Reply:
Thanks for pointing @watfordgnf! Working on the same.